src/test/java/io/spokestack/spokestack/webrtc/AutomaticGainControlTest.java

```diff
@@ -78,7 +78,7 @@ public void testProcessing() {
             .put("sample-rate", 8000)
             .put("frame-width", 10)
             .put("agc-target-level-dbfs", 9)
-            .put("agc-compression-gain-db", 2);
+            .put("agc-compression-gain-db", 15);
```

Comment:
Is it useful to list these last two anymore since they match the defaults?

Reply:
I think I'm okay with it in the test, and I couldn't actually get a bump in levels at 2, otherwise I wouldn't have touched it at all